integration-tests/test_kafka_to_redis.py

```diff
@@ -0,0 +1,37 @@
+import json
+import logging
+
+from common import kafka_utils, redis_utils
+from common.utils import run_job
+
+logger = logging.getLogger("dy")
+message_one = b'{"id":"1","name":"Boris"}'
+message_two = b'{"id":"2","name":"Ivan"}'
+
+
+def test_kafka_to_redis():
+    kafka_container = kafka_utils.get_kafka_container()
+    try:
+        with kafka_container as kafka:
```

Comment:
kafka_container will be terminated too after the run as redis' container? (for the latest, it's stopped in line 37(.

Reply:
Could you be more specific? Kafka container stops after with automatically. Or am I wrong? Start on beginning and stop at the end of scope